packages/api/src/services/library_item.ts

```diff
@@ -321,17 +321,16 @@ export const buildQueryString = (
               // return only deleted pages within 14 days
               return "(library_item.state = 'DELETED' AND library_item.deleted_at >= NOW() - INTERVAL '14 days')"
             default: {
-              let sql = 'library_item.archived_at IS NULL'
               if (useFolders) {
                 const param = `folder_${parameters.length}`
                 const folderSql = escapeQueryWithParameters(
                   `library_item.folder = :${param}`,
                   { [param]: value }
                 )
-                sql = `(${sql} AND ${folderSql})`
+                return `(library_item.archived_at IS NULL AND ${folderSql})`
```

Comment:
Did you want these changes in this PR too?

Reply:
Thanks! I will move these backend changes to a separate PR